templates/repo/diff/box.tmpl

```diff
@@ -230,17 +212,22 @@
 			document.addEventListener('DOMContentLoaded', function() {
 				$('tr.add-code').each(function() {
 					var prev = $(this).prev();
-					if(prev.is('.del-code') && prev.children().eq(3).text().trim() === '') {
-						while(prev.prev().is('.del-code') && prev.prev().children().eq(3).text().trim() === '') {
+					if(prev.is('.del-code') && prev.children().eq(5).text().trim() === '') {
+						while(prev.prev().is('.del-code') && prev.prev().children().eq(5).text().trim() === '') {
 							prev = prev.prev();
 						}
-						prev.children().eq(2).html($(this).children().eq(2).html());
+						prev.children().eq(3).attr("data-line-num", $(this).children().eq(3).attr("data-line-num"));
```

Comment:
If a line marker <td> is empty, it still changes the background color:

<img width="1036" alt="Screen Shot 2019-06-26 at 2 54 27 PM" src="https://user-images.githubusercontent.com/1669571/60206745-52faf800-9822-11e9-9fb6-fe29776f4ffe.png">

Is this code here what is messing with that? If the cause of that is obvious and seems quick to fix that would make it look nicer. There must be some similar logic that isn't changing the background of the code part when it is empty.

Reply:
Maybe @zeripath can have I look. I did not touch this code at all myself and would rather not want to 😉

Reply:
So I don't think that's the javascript - I think this is something to do with the css.

Reply:
So the empty code section for added lines should be grey not white. Another bug with my other PR.